test/addons/repl-domain-abort/test.js

```diff
@@ -54,7 +54,7 @@ dInput._read = function _read(size) {
 };
 
 dOutput._write = function _write(chunk, encoding, cb) {
-  if (chunk.toString().indexOf('cb_ran') === 0)
+  if (chunk.toString().startsWith('cb_ran'))
```

Comment:
I think it makes sense to decouple changes to `startsWith` to a different PR. As far as I can see from the discussion in https://github.com/nodejs/node/issues/12586, that PR won't be controversial like this one may potentially be :)